### DISCUSSION TITLE
Support for rendering italics?

### DISCUSSION THREAD

**Discussion by bschucha** [charmbracelet/glow#470] [General]
Am I just missing it somehow, or does glow currently not support rendering text wrapped in single stars ('*') or single underscores ('_') as italic?

**Comment by meowgorithm** [comment]
Hi! Glow does renders those as italics actually, so it may be a limitation or setting in your terminal and/or multiplexer. In particular I've found that tmux does not support italics out of the box.

You can check for italic support in your terminal with `echo`:

```
echo -e "\e[3mItalics\e[0m"
```

Often if italics are not supported you'll see text reversed instead.

I have the following in my `tmux.conf` for italic and truecolor support (though your mileage may vary here):

```
set -g default-terminal "xterm-256color"
set -ga terminal-overrides ",xterm-*:Tc"
```

**Reply by bschucha** [reply]
Excellent - that worked for me!  I now see italics in glow :)  Thanks for the quick and helpful response!
